html/changelogs/Kurfurst.yml

```diff
@@ -0,0 +1,14 @@
+author: Kurfurst
+delete-after: True
+changes:
+ - tweak: You can now repair the radio transmitter with a solder
+ - tweak: Radio transmitter is now dense (you can push it)
+ - tweak: Fixed a bug in which you couldn't examine someone's ear slot

```

Comment:
It really makes no difference to me which icon is used but yeah I am pretty sure based on the old code that this was a bug given that we did have examine text for ears, it just used the broken l_ear r_ear system instead of ears